Testscripts/Windows/DiskHotAddRemove-Parallel.ps1

```diff
@@ -60,17 +60,27 @@ function Main {
             throw "Fail to attach $count empty data disks of size $diskSizeinGB GB to VM"
         }
         Write-LogInfo "Verifying if data disks are added to the VM - running fdisk on remote VM"
-        $fdiskOutput = Run-LinuxCmd -username $user -password $password -ip $AllVMData.PublicIP -port $AllVMData.SSHPort -command "/sbin/fdisk -l | grep /dev/sd" -runAsSudo
-        foreach ($line in ($fdiskOutput.Split([Environment]::NewLine))) {
-            if ($line -imatch "Disk /dev/sd[a-z][a-z]|sd[c-z]:" -and [int64]($line.Split()[4]) -eq (([int64]($diskSizeinGB) * [int64]1073741824))){
-                $verifiedDiskCount += 1
-                Write-LogInfo "$verifiedDiskCount data disk is successfully attached the VM: $line"
+        $retry = 1
+        $retryMaxTimes = 30
```

Comment:
Please add a comment as why this retry is added.
Is there any particular observation to select 30*30

Reply:
actually, retry once, it will break.